scripts/scripts.js

```diff
@@ -144,6 +144,26 @@ async function decorateVideoLinks(main) {
   });
 }
 
+/**
+ * Decorates all multi-column sections in a container element.
+ * @param {Element} main The container element
+ */
+function decorateMultiColumnSections(main) {
+  main.querySelectorAll(':scope > div.column-section-1-1, :scope > div.column-section-3-2, :scope > div.column-section-2-3, :scope > div.column-section-2-1, :scope > div.column-section-1-2, :scope > div.column-section-3-1, :scope > div.column-section-1-3').forEach((section) => {
+    const left = document.createElement('div');
```

Comment:
Just a suggestion: we have been using the dom-builder.js to create DOM nodes and add CSS classes to them, as in: _div({ class: 'search-e' })_